models/schema.js

```diff
@@ -0,0 +1,56 @@
+const { Schema, model } = require("mongoose");
+const bCrypt = require("bcryptjs");
+
+const contacts = new Schema({
+  name: {
+    type: String,
+    required: [true, "Set name for contact"],
+  },
+  email: {
+    type: String,
+  },
+  phone: {
+    type: String,
+  },
+  favorite: {
+    type: Boolean,
+    default: false,
+  },
+  owner: {
+    type: Schema.Types.ObjectId,
+    ref: "user",
+  },
```

Comment:
Hej, to daj jeszcze obsługę po kluczu owner. To znaczy że kiedy szukasz kontaktu, to powinieneś szukasz teraz poprzez id i owner. Czyli że teraz będziesz miał zawsze dwa filtry jeżeli będziesz obsługiwał kontakty. Contact.findOne({_id(albo inny), owner});  `owner` to zawsze będzie ID użytkownika obecnego. Więc wszystkie ścieżki powinny uwierzytelniać użytkownika :) Ma sens?

Reply:
oczywiście , byłoby bez sensu gdyby każdy mógł widzieć wszystkie dane